controllers/tradeHistory.js

```diff
@@ -0,0 +1,22 @@
+const postHistory = async (req, res) => {
+	const trade_id = req.body.trade_id;
+
+	const tradeResult = await axios.post(
+		"https://sandbox.tappaysdk.com/tpc/transaction/trade-history",
+		{
+			partner_key:
+				"partner_wcxH5GX2HMLk9p2WpfOYSWseWOyn0mE0K1VwqjIZAgDGCsRZ4BEqMCaL",
+			rec_trade_id: trade_id,x
+		},
+		{
+			headers: {
+				"Content-Type": "application/json",
+				"x-api-key":
+					"partner_wcxH5GX2HMLk9p2WpfOYSWseWOyn0mE0K1VwqjIZAgDGCsRZ4BEqMCaL",
```

Comment:
i think you are forgetting this is a public repository 🌝
is it possible to move it to the .env file?

Reply:
Ya, I move it to .env file and have sent you the x-api-key.